bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/telescope/PosAngleConstraint.java

```diff
@@ -28,11 +29,6 @@ public String description() {
     /** The provided pos angle, or the pos angle plus 180 deg. */
     FIXED_180() {
         @Override
-        public ImList<Angle> steps(Angle start, Angle stepSize) {
```

Comment:
Ahhh... previously, the default was to run through the `stepSize` and calculate all angles, but since this behaviour only occurs in the `UNBOUNDED` case and now in three cases, the default is to use `start` and `start + 180`, I moved the previous default implementation into an overridden method in `UNBOUNDED` and made the `angle` and `angle.flip` the default behaviour.

Does that make sense? I think that explanation was a bit confusing. (Trying to fight cats from laying on my keyboard.)